Hello/Hello.cproject.yml

```diff
@@ -30,3 +30,4 @@ project:
     - component: Compiler:I/O:STDOUT&User
     - component: Device:Definition
     - component: Device:Startup&Baremetal
+    - component: ARM::CMSIS:OS Tick:SysTick@1.0.5

```

Comment:
Why should the version of the SysTick component be locked to 1.0.5
Adding the file `Hello.cbuild-pack.yml` would lock the ARM::CMSIS pack to version 6.0.0.

Reply:
No not really, I will set it to any version beyond 1.0.5

Reply:
Then we need to make this consistently for all components, right?

Reply:
I think ideally we should not mention the version if not required. I.e., only keep important information.